### DISCUSSION TITLE
Tool Resets Docuemnts, Downloads, Pictures, etc. Directories

### DISCUSSION THREAD

**Discussion by HarryBosch03** [ChrisTitusTech/winutil#1499] [General]
I had most of my user folders locations changed to a seccond drive, and after using the tool they changed back to their default location. Is that intended?

**Comment by sethro2** [comment]
I relocated Desktop, Documents, Pictures, Videos, etc. to a secondary drive as well on mine and the kids laptops.  Not a huge deal to fix but definitely noticed when disabling OneDrive.  There is a routine that copies down OneDrive folders and resets the folder locations assuming they were all linked to OneDrive.

Be neat to have a prompt to skip defaulting the user folders back to default.

**Reply by GrantRobertson** [reply]
Thank you for letting us know which feature reset these things. I had already uninstalled OneDrive, so I didn't really need to select that option, but I figured I would let it clean up any mess left behind by OneDrive. As it turns out, this utility made more of a mess than it cleaned up. However, overall, the utility is very helpful. As a former network manager, I used to have to do tons of these kinds of things on any new machine I set up. Granted, we used disk imaging for deploying that setup to all the other identical machines. But the main office was always buying slightly different machines, requiring fresh images to be set up. You know how it goes. 

Anyway, I will disable this tweak and not have to worry about it in the future.

**Comment by bulls45** [comment]
Have the same issue. I'm a sim racer and iRacing stores quite a lot of stuff in the documents folder: cars, tracks, setups, replays...I think there is more data there than in the actual installation folder and therefor my documents folder is on another drive. So after each time I used the tool to debloat the sim simply will not work anymore and I have to change paths of the documents folders again again. Is there an option to prevent this behavior of the tool or in fact an "issue"?
